apps/src/p5lab/P5Lab.js

```diff
@@ -322,6 +324,15 @@ P5Lab.prototype.init = function(config) {
   config.enableShowLinesCount = false;
 
   const onMount = () => {
+    try {
+      const localeCode = getScriptData('appoptions').locale;
```

Comment:
Have you tried using `window.appOptions.locale` instead? That's what P5LabView.jsx uses.